mi_analysis.m

```diff
@@ -23,8 +24,6 @@
         
         reparam %Do you reparameterize
         
-        k_audited = 'No' % Have the k-values been manually audited? Automatically set to yes once auditing function is run.
```

Comment:
I'm not sure if this got taken out of a particular version of the code, or if this was just not copied properly. But we do want this to be included.